### ISSUE TITLE
Pairwise tip distances

### ISSUE THREAD

**Issue by MichelleKendall** [thibautjombart/treescape#57]
Caroline pointed out that we could link "nNodes" and "patristic" with a parameter lambda, just like we do in our metric. Add this to Shiny app and make these methods more conveniently available from treescape function.

**Comment by thibautjombart** [comment]
Yep that's true, that would be a new metric.

**Comment by MichelleKendall** [comment]
Yes I think so - I will have to check the zero condition but I reckon the proof would hold by a very similar argument to the one we used for ours.
